app/mailers/smoke_test_prison_mailer.rb

```diff
@@ -0,0 +1,12 @@
+class SmokeTestPrisonMailer < PrisonMailer
+  def self.smtp_settings
+    {
+      address: ENV['SMOKE_TEST_SMTP_HOSTNAME'],
+      port: ENV['SMOKE_TEST_SMTP_PORT']
```

Comment:
We do. If you have a moment, it would be great if you could set that up, I think it'll be much quicker.